res/css/views/dialogs/_AddExistingToSpaceDialog.pcss

```diff
@@ -225,6 +225,7 @@ limitations under the License.
 .mx_AddExistingToSpace_entry {
     display: flex;
     margin-top: 12px;
+    width: 98%;
```

Comment:
Instead of using `98%`, I'd suggest using `calc(100% - [the actual amount of space required])`.

Reply:
@duxovni The scrollbar width is set to thin, the width would be different in different browsers and different zoom levels, that's why I set it to percentage and tested in different browsers with different zoom levels. I am unable to find a way to calculate the exact scrollbar width each time.

Reply:
We've solved this problem in the past in the room list and such - applying the same solution here would probably work.

Reply:
I have pushed the required changes @duxovni